_includes/ont_pools.md

```diff
@@ -0,0 +1,49 @@
+<a name="pools" href="#" id="toplink">top</a>
+
+# 4. Pooling Libraries
+
+A library cannot be directly loaded into a _flow cell_. A _dilution_ must be made, and
+then many dilutions (or just one) can be mixed into a _pool_ for sequencing.
+
+_Orders_ are requests for sequencing pools a certain number of times. They are used to
+keep track of sequencing progress for project management and book-keeping.
+
+## 4.1 Creating a Dilution
+
+1. If you are still on the Create Libraries page, click the _Make dilutions_ button in the
+toolbar. Otherwise, go to the Libraries page, select the library you created in
+the previous exercise, and click the _Make dilutions_ button.
+1. Click the _Create_ button in the dialog that pops up.
+1. Enter the dilution information:
+  * _Conc._: Enter a concentration.
+  * _Creation Date_: Select the current date.
```

Comment:
For this and all other 'Creation Date' steps, mine was pre-selected. Is this normal?